File: RepoTransBench/source_projects/Python/serkanyersen_underscore.py/tests/test_arrays.py
import sys
import os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
from src import underscore

def test_chunk():
    assert underscore.chunk([1, 2, 3, 4], 2) == [[1, 2], [3, 4]]

def test_compact():
    assert underscore.compact([0, 1, False, 2, '', 3]) == [1, 2, 3]

File: RepoTransBench/target_projects/Python/Rust/serkanyersen_underscore.py/tests/test_arrays.rs
use serkanyersen_underscore::underscore::{chunk, compact, Compactable};

#[test]
fn test_chunk() {
    let v = vec![1, 2, 3, 4];
    let expected = vec![vec![1, 2], vec![3, 4]];
    assert_eq!(chunk(&v, 2), expected);
}

#[test]
fn test_compact() {
    use Compactable::*;
    let arr = vec![Int(0), Int(1), Bool(false), Int(2), String("".into()), Int(3)];
    let expected = vec![Int(1), Int(2), Int(3)];
    assert_eq!(compact(&arr), expected);
}